processes.go

```diff
@@ -129,7 +159,7 @@ func (cm CommandMap) Value() (driver.Value, error) {
 	for k, v := range cm {
 		m[string(k)] = sql.NullString{
 			Valid:  true,
-			String: string(v),
+			String: v.String(),
```

Comment:
Dang, just realized that this is a problem. By stringifying it here, we lose information.

It would technically be fixed with https://github.com/remind101/empire/pull/792, but might be worth fixing in a separate PR so we can roll out the recent migrations more incrementally to check for problems.